build.sbt

```diff
@@ -1221,7 +1221,7 @@ def archetypeProject(archetypeName: String) =
         (unmanagedResources in Compile).value ++ gitIgnoreFiles
       },
       // Don't force copyright headers in Maven archetypes
-      excludeFilter.in(headerSources) := "*"
```

Comment:
According to [the docs](https://github.com/sbt/sbt-header#excluding-files), this should have worked.